src/Sylius/Component/User/composer.json

```
@@ -67,7 +66,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Sylius\\Component\\User\\spec\\": "spec/"
+            "Tests\\Sylius\\Component\\User\\": "Tests/"
```

I would be for using lowercase for tests directory, especially in components/bundles where we are introducing this directory
```suggestion
            "Tests\\Sylius\\Component\\User\\": "tests/"
```